src/popup/pages/SignTransaction.vue

```diff
@@ -51,8 +58,24 @@ export default defineComponent({
           return;
         }
 
+        const replaceCallerTx = async (tx: Encoded.Transaction) => {
+          const unpackedTx = unpackTx(tx);
+          if (unpackedTx.tag === Tag.ContractCallTx || unpackedTx.tag === Tag.ContractCreateTx) {
+            // TxUnpacked isn't exported
+            // typescript is unaware that ContractCallTx and ContractCreateTx do have a callerId
+            (unpackedTx as any & { callerId: Encoded.AccountAddress })
+              .callerId = activeAccount?.address as Encoded.AccountAddress;
+            unpackedTx.nonce = (await aeSdk.api.getAccountByPubkey(
+              activeAccount?.address as Encoded.AccountAddress,
```

Comment:
This is already used before. Maybe you can create constant first:
```
const activeAccountAddress = activeAccount?.address as Encoded.AccountAddress;
```

Reply:
agree, will adjust

Reply:
adjusted